assets/scss/modules/_notifications.scss

```diff
@@ -163,7 +163,8 @@
 }
 
 .highlight-message--icon-top {
-  background-position-y: em(20);
+  background-position: em(20) em(20);
+  background-size: em(27) em(27);
```

Comment:
Also you may have problems with em() on certain browsers here it would be better to go with `px` values